mod_app/templates/film_detail.html

```diff
@@ -5,8 +5,165 @@
     <div>
         <h1>{{film.title}}</h1>
         <p>{{film.alt_titles}}</p>
+        <p>{{film.release_date}}</p>
 
     </div>
 
+    <div class="tab-container">
+
+        <div class="tab-titles">
+
+            <label class="tab-title" for="filmic-tab">Filmic Information
+                <input type="radio" id="filmic-tab" name="tab-group" checked>
+            </label>
+
+
+            <label class="tab-title" for="technical-tab">Technical
+                <input type="radio" id="technical-tab" name="tab-group">
+            </label>
+
+
+            <label class="tab-title" for="nonfilmic-tab">Non Filmic
+                <input type="radio" id="nonfilmic-tab" name="tab-group">
+            </label>
+
+            <label class="tab-title" for="bibliography-tab">Bibliography
+                <input type="radio" id="bibliography-tab" name="tab-group">
+            </label>
```

Comment:
These tabs titles are visible when the page first loads, but only the selected one is visible once I click on one of them.

Reply:
That might be a browser-specific problem but also good to know. It works on chrome, brave and safari, which browser are you seeing the issue with?

Reply:
Firefox

Reply:
Just tried it on Chrome: non-selected tabs are visible, but still very faint. Also the film list table looks waaay better on Chrome.